cubi_isa_templates/isatab-stem_cell_core_sc/cookiecutter.json

```diff
@@ -1,7 +1,7 @@
 {
   "group": "CoreRnD",
   "study_title": "Study Title",
-  "sample_names": "sample1,sample2",
+  "source_names": "sample1,sample2",
```

Comment:
This actually is the sample name and not the source name

Reply:
It's the source_name variable being re-used to define sample names. Before it was the sample_name variable being re-used to define source names.

Reply:
You're right. Reverted this change.